.gitignore

```diff
@@ -32,9 +32,14 @@ src/javasource/system/UserActionsRegistrar.java
 /build/
 private/cache/retriever/catalog.xml
 package.xml
-/src/CommunityCommons/userlib/
-/src/CommunityCommons/build/
 *.mda
+/src/CommunityCommons/build/
+/src/CommunityCommons/javasource/*/proxies/

```

Comment:
Can you ignore `/src/CommunityCommons/javasource/system/` as well? Everything from the System module is generated, right?